cypress/support/createUtils.js

```diff
@@ -13,7 +13,7 @@ import { ITEM_TYPES } from '../fixtures/enums';
 
 // eslint-disable-next-line import/prefer-default-export
 export const createItem = (payload, options) => {
-  cy.get(`#${CREATE_ITEM_BUTTON_ID}`).click();
+  cy.get(`#${CREATE_ITEM_BUTTON_ID}`).click({ force: true });
```

Comment:
Why is this forced? Maybe add a comment

Reply:
I added force, because tooltips can sometimes block the view, it looks like buttons with tooltips are kept in hover state even after clicking with cypress. Using force ensures that the click will go through even if the element is partially blocked by a tooltip.